### PR TITLE
Add dbjson 

### LINKED ISSUE #4: Setup db.json (1)
(2) -> MongoDB and ExpressJS

### Notes/Links
Install JSON-Server
npx json-server --watch db.json --port 5000

Use placeholder code.

```
{
  "posts": [{ "id": 1, "title": "json-server", "author": "typicode" }],
  "comments": [{ "id": 1, "body": "some comment", "postId": 1 }],
  "profile": { "name": "typicode" }
}
```

### Checkboxes

- [ ] Install JSON-Server (npx json-server --watch db.json --port 5000)
- [ ] User placeholder code

### PR DESCRIPTION
**Pull request by Naasgul**
closes #4 

We installed the json-server to test some basic backend functionality. In the end we used the local-storage to greet the user personally.

reviews appreciated!

### CODE REVIEW

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -32,14 +36,38 @@ const LoginContainer = styled.div`
   }
 `;
 
+function setLS(name) {
```
Comment (kschmitz123) [inline client/src/pages/Login.js:39]: i would change the naming here

**Comment by kschmitz123** [comment]
lgtm 🚀

File: client/src/api/users.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const response = await fetch(`http://localhost:5000/users/`, {
+    method: "POST",
+    body: JSON.stringify(name),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  const newName = await response.json();
+  return newName;
+}
```
Comment (AlexisRoe) [inline client/src/api/users.js:23]: you should use a constant instead, for not screwing it with typos 😉

```suggestion

const DB_JSON = "http://localhost:5000"

export async function getUserById(id) {
  const response = await fetch(`${DB_JSON}/users/${id}`);
  const user = await response.json();
  return user;
}

export async function getUsers() {
  const response = await fetch(`${DB_JSON}/users`);
  const users = await response.json();
  return users;
}

export async function postUser(name) {
  const response = await fetch(`${DB_JSON}/users/`, {
    method: "POST",
    body: JSON.stringify(name),
    headers: {
      "Content-Type": "application/json",
    },
  });
  const newName = await response.json();
  return newName;
}
```

**Review by AlexisRoe — COMMENTED** [review]
👍

File: client/src/api/users.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+export async function getUserById(id) {
+  const response = await fetch(`http://localhost:5000/users/${id}`);
```
Comment (phlgr) [inline client/src/api/users.js:2]: Those paths are not relative and will not work in production

File: client/src/api/users.js (hunk 2 above)
Comment (phlgr) [reply client/src/api/users.js:23]: No, this should not be saved in a constant. These paths should all be relative

File: package.json [hunk 4]
```diff
@@ -16,7 +16,8 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "client": "cd client && npm start",
     "server": "nodemon server.js",
-    "start": "node server.js"
+    "start": "node server.js",
+    "api": "cd client && npx json-server --watch db.json --port 5000"
```
Comment (phlgr) [inline package.json:20]: I would suggest you install json-server and implement it into your server.js
https://www.npmjs.com/package/json-server#module

**Review by phlgr — COMMENTED** [review]
Please resolve the warnings before merging next time.

File: client/src/api/users.js (hunk 2 above)
Comment (AlexisRoe) [reply client/src/api/users.js:23]: alright sorry

**Pull request merged**